### ISSUE TITLE
zotero Add-ons dialog not opening

### ISSUE THREAD

**Issue by ebw44** [scitedotai/scite-zotero-plugin#31]
# Description
I believe that this plugin is preventing me from opening the 'Add-ons' dialog (from 'Tools/Add-ons'). I cannot be 100% sure it is the last plugin I have installed, and as I did the installation through that dialog, it was working before. Also the 'Error Console' shows that message when trying to open the dialog:  
`No chrome package registered for chrome://zotero-better-bibtex/skin/scite.svg`

# Installation
Zotero 5.0.97-beta.57+07df7d0de
scite 1.11.1

**Comment by u-ashish** [comment]
Thanks for reporting this @ebw44 -- I tried just now with the same build of Zotero and our plugin and while I do see that error message (I can fix that separately), I am able to open the Add-ons dropdown.

Could you share a screenshot of the Error console so I can see what else might be interfering, or let me know what other plugins are installed so I can reproduce better?

**Comment by ebw44** [comment]
Here is a screenshot after opening zotero and trying to open the 'Add-ons' dialog twice. 
![image](https://user-images.githubusercontent.com/8319368/144517061-7960a76e-4306-4098-9c34-b3023694a514.png)

PS: I'm not able to do copy in the 'Error Console'

**Comment by u-ashish** [comment]
Yeah no worries about copy paste. Are you able to try on a fresh installation of Zotero, with no plugins, and then try with just ours, and see if it still has the same issue?

**Comment by ebw44** [comment]
I was able to remove/disable all the add-ons (thanks to https://forums.zotero.org/discussion/85974/zotero-plugin-trouble#latest). At that point I was able to open the dialog and remove scite add-ons. And then everything seems to work fine.

The second test I run is to disable all the add-ons, restart Zotero and install scite. At that point I still get the same `scite.svg` message but the dialog opens.
From there I reenable all my add-ons in the following order: Zotfile, Zotutil, Cita, DOIManager. With a restart of zotero everytime, and everything was and is working fine.

I believed I had a Zotero process hanging (without any windows) before (without any windows) which may have cause the issue of the add-ons dialog not opening. So happy to close that issue.
